src/Components/OpiumTooltip/index.tsx

```diff
@@ -107,16 +108,19 @@ const OpiumTooltip: FC<Props> = (props: Props) => {
       }
     >
       {
-        component
-          ? component
-          : <Button
-            variant={'primary'}
-            label={label}
-            onClick={() => { }}
-            style={componentStyles}
-            onMouseEnter={() => setHover(true)}
-            onMouseLeave={() => setHover(false)}
-          />
+        disabled ? (
```

Comment:
I'd rather do it somehow else.
Maybe wrap all disabled buttons in div.
Or maybe solve it just with css tooltip > button {pointer-events} if this is the issue. Because we will add only disabled buttons in tooltip. 
The point is, it's not obvious that you have to pass "disabled" prop to tooltip to make it work on disabled buttons, need to do it more natively